README.md

```diff
@@ -40,12 +40,6 @@ If you are running Notebook 5.2 or earlier, enable the server extension by runni
 jupyter serverextension enable --sys-prefix jupyterlab_latex
 ```
 
-To install the lab extension, run
```

Comment:
While dropping this is good, i often add a line about `jupyter labextension list` and what one should expect to see.

Reply:
Added some instructions as suggested